wikipedia-dark.user.css

```diff
@@ -645,6 +645,176 @@ regexp("https?:\/\/wiki\.(archlinux|mozilla)\.(org|jp)\/.*$") {
+  td[bgcolor="#eeeeee"],
+  tr[bgcolor="#eeeeee"] > td {
+    background: var(--gray-3) !important;
+  }
+  table[class="toccolours;"] tr[style*="background:#FAFAFA"],
+  .ve-ui-cxDesktopContext .ve-ui-cxLinkContextItem-sourceBody {
+    background: var(--gray-4) !important;
+  }
+  .infobox[style^="background:#F5F5F5;"],
+  .mw-parser-output > table.vertical-navbox,
+  #uls-settings-block,
+  .mw-parser-output td[style*="background-color:#efefef;"],
+  .mw-cx-ui-CategoryTagItemWidget.oo-ui-flaggedElement-highlight {
+    background: var(--gray-18) !important;
+  }
+  input.languagefilter {
+    background: var(--gray-2);
+  }
+  tr td[style="background: #FFFFFF; color:#000000;"],
+  .cx-category-listing.oo-ui-icon-tag {
+    color: var(--black) !important;
+  }
+  .userbox tbody tr td,
+  .frb-headline > span,
+  .infobox td.infobox-full-data th.infobox-above,
+  .infobox-above[style*="background-color:#b0c4de"],
+  .infobox-above[style*="background-color:#FFD"],
+  .infobox-above[style*="background-color: #91FAFA"],
+  .infobox-above[style="background-color: #b0c4de"],
+  .oo-ui-icon-tag,
+  .cx-message-widget-message,
+  td[style^="background:  #FFFF"],
+  td[style^="background:  #DFDFDF;"],
+  td[style*="background:  #CFCFFF;"],
+  td[style*="background:  #EFCFFF;"],
+  td[style*="background:  #DFFFDF;"],
+  td[style*="background:  #FFDF9F;"],
+  tr > td[style="background: #FFFFFF;"],
+  tr > td[style="background: #CFE8FF;"],
+  tr > td[style="background: #EED8AE;"],
+  tr > td[style="background: #FFF8DC;"],
+  tr > td[style="background: #B9D3FF;"],
+  tr > td[style="background: #6495ED;"],
+  tr > td[style="background: #607CD2;"],
+  tr > td[style="background: #828BD9;"],
+  tr > td[style="background: #F0F8FF;"] {
+    color: var(--gray-1) !important;
+  }
+  .reference-text span[style^="color:#555"] {
+    color: var(--gray-a) !important;
+  }
+  .userbox tr td,
+  .frb-headline > span,
+  .frb-inline-message,
+  .tux-pagemode-translation,
+  .cx-translator__total-translations,
+  .cx-translator__header,
+  .tux-editor-insert-buttons button,
+  .tux-proofread-translation,
+  .tux-proofread-source,
+  .cx-tlitem__details .target-title,
+  .ve-ui-cxLinkContextItem-language,
+  .cx-header-draft-status,
+  .oo-ui-labelElement-label,
+  .ve-cx-toolbar-mt-title {
+    color: var(--gray-c) !important;
+  }
+  .cx-tlitem .cx-tlitem__details .last-updated,
+  .cx-tlitem__languages__language,
+  .cx-slitem__languages__language,
+  .cx-selected-source-page__license,
+  .mw-cx-tools-IssueTracking > .mw-cx-tools-IssueTracking-head .oo-ui-labelWidget,
+  .cx-column--language-label,
+  .cx-column-language-label {
+    color: var(--gray-c);
+  }
+  .cx-header__trademark-text,
+  .cx-translationlist__header {
+    color: var(--white);
+  }
+  .cx-translationlist,
+  .cx-dashboard-sidebar__help,
+  .cx-translator {
+    box-shadow: 0 1px 1px var(--gray-5);
+  }
+  .tux-message-editor .messagekey .caret {
+    border-top: 4px solid var(--gray-c);
```

Comment:
Further on the `border` property, if the only thing you're changing is the color (which should be the case 99% of the time), you should use just `border-color` here, or when necessary, one of the `border-*-color` properties.

So in this case, if you're only modifying color, it would be:
```
border-top-color: var(--gray-c);
```

Reply:
Nice catch, forgot about it.